src/main/java/de/tum/cit/aet/artemis/quiz/web/QuizExerciseCreationUpdateResource.java

```diff
@@ -75,15 +73,18 @@ public class QuizExerciseCreationUpdateResource {
 
     private final ExerciseVersionService exerciseVersionService;
 
+    @Value("${jhipster.clientApp.name}")
+    private String applicationName;
```

Comment:
Can we please use instructor injection here? https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/server.html#dependency-injection